### PR TITLE
Update docs with device tagging command

### PR DESCRIPTION
**Pull request by MahrRah**
## iotedgedev Pull Request Checklist

- [ ] Versions updated (both __init__.py files, CHANGELOG, setup.py, setup.cfg)
- [ ] Unit tests pass locally
- [ ] Quickstart steps locally validated
- [ ] Passes unit tests in CICD pipeline (green on Github pipeline)
- [ ] Pypi RC version passes Edge CICD pipeline validation for cross-tool validation

## Additional comments

### CODE REVIEW

File: docs/Command-List.md [hunk 1]
```diff
@@ -542,3 +546,19 @@ Options:
...
   -h, --help           Show this message and exit.
 ```
+
+
+## iotedgedev solution tag
+
+```text
+Usage: iotedgedev solution tag [OPTIONS]
+
+  Adds tags to device twin for layered deployments
```
Comment (marianan) [inline docs/Command-List.md:556]: >layered [](http://example.com/codeflow?start=31&length=7)

should it say automated deployments instead?

Comment (EliiseS) [reply docs/Command-List.md:556]:
```suggestion
  Adds tags to device twin for automatic deployments
```

Comment (EliiseS) [reply docs/Command-List.md]: I've added additional changes for this PR here: #507 
(I'm unable to change new files in this PR due to lack of permissions on Tamaras fork)

**Review by marianan — APPROVED** [review]
:shipit:

**Pull request merged**